Resources/public/js/apps/ez-platformuiapp.js

```diff
@@ -156,6 +170,50 @@ YUI.add('ez-platformuiapp', function (Y) {
         },
 
         /**
+         * Add or deletes handleSideViews and handleMainViews from the routes callbacks
+         *
+         * @method _alterRoutesCallback
+         * @protected
+         */
+        _alterRoutesCallback: function() {
+            var app = this;
+
+            Y.Object.each(app.get('routes'), function(route) {
+                var index = route.callbacks.indexOf('handleSideViews');
```

Comment:
that's a bad condition, actually each and every route should be disabled somehow. Most likely you need to store somewhere the initial `callbacks` values and restore them afterwards.